server/pkg/api/accountHandlers.go

```diff
@@ -6,16 +6,78 @@ import (
 
 	"github.com/darwinfroese/hacksite/server/models"
 	"github.com/darwinfroese/hacksite/server/pkg/accounts"
+	"github.com/darwinfroese/hacksite/server/pkg/auth"
 )
 
-var accountHandlerMap = map[string]handler{
+var accountsHandlerMap = map[string]handler{
```

Comment:
These can be combined: with POST being used for create, PUT and PATCH both being used for update and GET for getting the account

Reply:
ok will combine them, added a new handler because the issue suggested `/account` not `/accounts` :+1:

Reply:
Ah, that makes sense. 

Ya it should combine them into `accounts`. That's my fault!